tuist/tuist
# Add support for reading environment variables from the`Tuist/.env` file

Resolves https://github.com/tuist/tuist/issues/4139

### Short description 📝

Implements `.env` file support for loading environment variables into Tuist. 

### How to test the changes locally 🧐

- Create a `.env` file inside the Tuist directory
- add env vars prefixed with `TUIST_`
- add a check in your manifest that fails if the environment variable is not present
- run any tuist command

### Checklist ✅
